src/ch/imedias/rsccfx/view/RsccHomeView.java

```diff
@@ -45,31 +61,49 @@ public RsccHomeView(Rscc model) {
     initFieldData();
     layoutForm();
     bindFieldsToModel();
+
   }
 
   private void initFieldData() {
-    // populate fields which require initial data
-    requestViewBtn.textProperty().setValue("I need help\nGet somebody to help you remotely");
-    supportViewBtn.textProperty().setValue("I want to assist someone\nSomebody needs my help");
+    requestTextBig = new Label("I need help");
+    requestTextSmall = new Label("Get somebody to help you remotely");
+    supportTextBig = new Label("I want to assist someone");
+    supportTextSmall = new Label("Somebody needs my help");
   }
 
   private void layoutForm() {
+
+    requestTextBig.setFont(new Font(35));
```

Comment:
please change these in the CSS, else we don't have control over what font is used